src/boot/efi/boot.c

```diff
@@ -618,6 +622,53 @@ static EFI_STATUS reboot_into_firmware(void) {
         assert_not_reached();
 }
 
+static bool entry_is_visible(ConfigEntry *entry, ConfigEntry *visible_group) {
+        assert(entry);
+
+        if (entry->group_entry == NULL)
+                return true;
+        else if (entry->group_entry == visible_group)
```

Comment:
Please drop the `else` here… It doesn't matter much, but that's the usual coding style.

Reply:
done